engine/Default/course_plot_result.php

```diff
@@ -12,11 +12,7 @@
 $next_sector = $path->getNextOnPath();
 
 if ($player->getSector()->isLinked($next_sector)) {
-
-	// save this to db (if we still have something)
-	if ($path->getTotalSectors()>0) {
 		$player->setPlottedCourse($path);
```

Comment:
You didn't unindent this when you moved it out of the `if` so it's now doubly indented

Reply:
Thanks -- fixed!